.github/workflows/release_kolibri.yml

```diff
@@ -0,0 +1,112 @@
+name: Release Kolibri
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  whl:
+    name: Build WHL file
+    uses: ./.github/workflows/build_whl.yml
+  pex:
+    name: Build PEX file
+    needs: whl
+    uses: ./.github/workflows/build_pex.yml
+    with:
+      whl-file-name: ${{ needs.whl.outputs.whl-file-name }}
+  dmg:
+    name: Build DMG file
+    needs: whl
+    uses: learningequality/kolibri-app/.github/workflows/build_mac.yml@main
+    with:
+      whl-file-name: ${{ needs.whl.outputs.whl-file-name }}
+      release: true
+      ref: main
+    secrets:
+      KOLIBRI_MAC_APP_IDENTITY: ${{ secrets.KOLIBRI_MAC_APP_IDENTITY }}
+      KOLIBRI_MAC_APP_CERTIFICATE: ${{ secrets.KOLIBRI_MAC_APP_CERTIFICATE }}
+      KOLIBRI_MAC_APP_CERTIFICATE_PASSWORD: ${{ secrets.KOLIBRI_MAC_APP_CERTIFICATE_PASSWORD }}
+      KOLIBRI_MAC_APP_USERNAME: ${{ secrets.KOLIBRI_MAC_APP_USERNAME }}
+      KOLIBRI_MAC_APP_PASSWORD: ${{ secrets.KOLIBRI_MAC_APP_PASSWORD }}
+  deb:
+    name: Build DEB file
+    needs: whl
+    uses: learningequality/kolibri-installer-debian/.github/workflows/build_deb.yml@master
+    with:
+      tar-file-name: ${{ needs.whl.outputs.tar-file-name }}
+      ref: master
+  exe:
+    name: Build EXE file
+    needs: whl
+    uses: learningequality/kolibri-installer-windows/.github/workflows/build_exe.yml@develop
+    with:
+      whl-file-name: ${{ needs.whl.outputs.whl-file-name }}
+      ref: develop
+  zip:
+    name: Build Raspberry Pi Image
+    needs: deb
+    uses: learningequality/pi-gen/.github/workflows/build_zip.yml@master
+    with:
+      deb-file-name: ${{ needs.deb.outputs.deb-file-name }}
+      release: true
+      ref: master
+  test_pypi_upload:
+    name: Upload to TestPyPi
+    needs: whl
+    uses: ./.github/workflows/pypi_upload.yml
+    with:
+      whl-file-name: ${{ needs.whl.outputs.whl-file-name }}
+      tar-file-name: ${{ needs.whl.outputs.tar-file-name }}
+      test: true
+  deployment_block:
```

Comment:
A bit nit picky, but maybe this should be renamed? At first I read this as "the block of code that will deploy". Then when I read the description, it seems like it is _blocking_ deployment. But then the steps is "Release now publishing!"... so, I'm not 100% sure what exactly this should be handling and if the feedback in the logs will be clear.

Reply:
Ah, yes - this is meant to be `the job that blocks deployment of the release` - thus requiring further manual intervention to make the release go.

The intention here is that we can test all the assets for the release, and then manually say "Good to go", and let the rest happen.

The reason that the only step is `Release now publishing!` is that this step will not run until it has been approved, so that message would only appear in the logs once we were going forward with the release.